packages/api/src/SignerPayload.ts

```diff
@@ -52,7 +52,10 @@ export default class SignerPayload extends Struct.with({
       method: method.toHex(),
       nonce: nonce.toHex(),
       tip: tip.toHex(),
-      version: version.toNumber()
+      version: version.toNumber(),
+      get encoded (): Uint8Array {
```

Comment:
Effectively we now have the same info in the same payload - twice. This doesn't quite work.